cowprotocol/cowswap
# fix(limit-orders): remove high price impact warning flickering

# Summary

Fixes #3212

The cause of the problem is quite banal: the warning showing flag was depending on quote loading value.

<img width="573" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/e6f200b6-4428-447b-8c44-d2326d0bfedb">

The issue with **unknown** price impact flickering was fixed [here](https://github.com/cowprotocol/cowswap/pull/3418).

  # To Test

1. Set some trade in Limit orders
2. Input very low price 
- [ ] "Your limit price is xx% lower than current market price" warning is displayed
3. Wait until the next quote update
- [ ] Nose picking cow is shown
- [ ] The warning doesn't flick (both in the limit order page and on its confirm modal)
